docs/docsite/rst/os_guide/windows_dsc.rst

```diff
@@ -205,10 +205,10 @@ For example, to define a ``[CimInstance]`` value in Ansible:
 
     # [CimInstance]AuthenticationInfo == MSFT_xWebAuthenticationInformation
     AuthenticationInfo:
-      Anonymous: no
-      Basic: yes
-      Digest: no
-      Windows: yes
+      Anonymous: false
```

Comment:
These values are for Windows-specific configuration and are not related to Ansible. Please check respective Windows documentation if these values accept True/False.

Reply:
Noted. As per some relevant code here, it seems Windows supports true/false, let me know what you think.

https://github.com/dsccommunity/WebAdministrationDsc/blob/e479b30fb9de6bd6778a98adfad20a3b3401210f/source/DSCResources/DSC_WebApplication/DSC_WebApplication.psm1#L590

Reply:
ok